Formula/sslyze.rb

```diff
@@ -12,53 +27,62 @@ class Sslyze < Formula
     sha256 "0fab1f496b3f52e7637e0ed8ef23d8b435b925150ca15b85899c95cf61084e8b" => :mavericks
   end
 
-  depends_on :arch => :x86_64
-  depends_on :python if MacOS.version <= :snow_leopard
+  head do
+    url "https://github.com/nabla-c0d3/sslyze.git"
 
-  resource "nassl" do
-    url "https://github.com/nabla-c0d3/nassl/archive/v0.12.tar.gz"
-    sha256 "40b3766fe98144e912ea7a8f1c34bf974e76f99ac40f128c3ce50b46b1fe315e"
-  end
+    # pypi version has prebuilt binaries
+    resource "nassl" do
+      url "https://github.com/nabla-c0d3/nassl.git"
+    end
 
-  resource "openssl" do
-    url "https://www.openssl.org/source/old/1.0.2/openssl-1.0.2d.tar.gz"
-    sha256 "671c36487785628a703374c652ad2cebea45fa920ae5681515df25d9f2c9a8c8"
+    resource "openssl" do
+      url "https://github.com/PeterMosmans/openssl.git", :branch => "1.0.2-chacha"
+    end
   end
 
+  depends_on :arch => :x86_64
+  depends_on :python if MacOS.version <= :snow_leopard
+
   resource "zlib" do
-    url "http://zlib.net/zlib-1.2.8.tar.gz"
-    sha256 "36658cb768a54c1d4dec43c3116c27ed893e88b02ecfcb44f2166f9c0b7f2a0d"
+    url "http://http.debian.net/debian/pool/main/z/zlib/zlib_1.2.8.dfsg.orig.tar.gz"
```

Comment:
Bleh. I keep meaning to audit this to death. Let's stick to the secure Debian mirrors 🙈.